src/hfx_libint_wrapper.F

```diff
@@ -66,6 +69,10 @@ MODULE hfx_libint_wrapper
 #endif
    END TYPE
 
+   !This is a safeguard/hack because 3-center ERIs are bugged on libint 2.5
```

Comment:
what is the nature of the safeguard here? Can it be removed at some point?

Reply:
This is a bug in libint 2.5 and resolved in libint 2.6. In order to preserve backwards compatibility with libint 2.5 (which was the first version of libint 2 compatible with cp2k), I suggested this solution. So we can remove it once we feel that there is no need to support libint 2.5.

Reply:
This is also linked to your previous note about the commented `libint2_build_3eri` at the beginning of the file. In principle it should be imported, but it can be used directly if coming from libint 2.5

Reply:
I've talked to @pseewald about that: for the CP2K v7.1 release we should drop that completely since it will be the first release requiring libint-2, at which point we a requirement for a minimal version is reasonable.

What is needed here is a description of the workaround, something like: `manual declaration of libint2_build_3eri function since declaration provided by the Fortran bindings in libint-2.5.0 are wrong (fixed in libint-2.6+)`